docs/cave/d2d-gis-shapefiles.md

```diff
@@ -5,10 +5,12 @@ This section describes how to start the GIS application, configure the display a
 
 ## Display GIS Data
 
-The GIS application is accessed through the CAVE menu by first selecting the Import option, and then choosing GIS Data from the Import submenu. The GIS DataStore Parameters dialog is comprised of four sections:
+The GIS application is accessed through **File > Import > GIS Data**. The GIS DataStore Parameters dialog is comprised of four sections:
+
+<center>![image](../images/gis-gui.png)</center>
```

Comment:
For some reason when viewing this page in github the image isn't showing up...but the file is there and it renders on my localhost